tests/e2e/support/api/graph/userManagement.ts

```diff
@@ -160,3 +160,17 @@ export const assignRole = async (admin: User, id: string, role: string): Promise
   })
   checkResponseStatus(response, 'Failed while assigning role to the user')
 }
+
+interface GroupResponse {
+  value: Group[]
+}
+
+export const getGroups = async (admin: User): Promise<Group[]> => {
```

Comment:
Here I see that variable name is `admin`, but 
```
const storeKeycloakGroups = async (user: User, usersEnvironment) => {
  const groups = await api.graph.getGroups(user)
```
The function that calls this takes the user as a parameter. Is the storage done only by the admin user? If that's the case, could the variable name be `adminUser`?

Reply:
i think getGroup can be done by anyone despite the ocis user role

Reply:
sorry i was confused by past merged code  
```
 if (!config.basicAuth) {
    const user = this.usersEnvironment.getUser({ key: 'admin' })
    if (config.keycloak) {
      await setAccessTokenForKeycloakOcisUser(user)
      await setAccessTokenForKeycloakUser(user)
      await storeKeycloakGroups(user, this.usersEnvironment)
```
as you can see, even though the user is **admin**, **user** variable name is being used 😄